src/store/actions/loadStateActions.ts

```diff
@@ -0,0 +1,38 @@
+export enum reducerDomains {
+    AUTH,
+    PROFILE,
+    TASKS,
+    CALENDAR,
+}
+
+export enum loadStateActionTypes {
+    LOADED,
+    LOADING,
+    ERROR,
+    ERRORHANDLED,
+}
+
+export const AuthStateActions = mapLoadStateActions(reducerDomains.AUTH)();
```

Comment:
Since this is essentially all load states types of actions, why don't we name them ${type}LoadStateActions